### ISSUE TITLE
`inc` command seems to not exists, the only example from docs fails

### ISSUE THREAD

**Issue by V0ldek** [nushell/nushell#7820]
### Describe the bug

The first example in the book on [Pipelines](https://www.nushell.sh/book/pipelines.html) contains the `inc` command:

```nu
> open "Cargo.toml" | inc package.version --minor | save "Cargo_new.toml"
```

I am trying to do _exactly_ that with my Cargo.toml, but it fails:

```nu
〉open "Cargo.toml" | inc package.version --minor | save "Cargo_new.toml"                                                                01/21/2023 07:30:16 PM
Error: nu::shell::external_command (link)

  × External command failed
   ╭─[entry #11:1:1]
 1 │ open "Cargo.toml" | inc package.version --minor | save "Cargo_new.toml"
   ·                     ─┬─
   ·                      ╰── executable was not found
   ╰────
  help: No such file or directory (os error 2)
```

The `Cargo.toml` is there, in particular getting the version behaves normally:

```nu
〉open "Cargo.toml" | get package.version
0.2.0
```

Moreover, `which inc` returns an empty list and `help inc` gives a Not found error:

```nu
〉help inc
Error: nu::parser::not_found (link)

  × Not found.
   ╭─[entry #14:1:1]
 1 │ help inc
   ·      ─┬─
   ·       ╰── did not find anything under this name
   ╰────
```

### How to reproduce

1. Create any file with a semver version in it, for example by doing `cargo new repro; cd repro`
2. Run `open "Cargo.toml" | get package.version` to verify that the file exists and has the correct field.
3. Try running `open "Cargo.toml" | inc package.version --minor`.

### Expected behavior

On a Cargo.toml with package version 0.1.0 the expected result would be 0.1.1.

### Screenshots

![image](https://user-images.githubusercontent.com/20441980/213884131-70aab8b3-0bf9-46c1-a6f1-d1f2f27db10f.png)

### Configuration

| key                | value                                |
| ------------------ | ------------------------------------ |
| version            | 0.74.0                               |
| branch             |                                      |
| commit_hash        |                                      |
| build_os           | linux-x86_64                         |
| build_target       | x86_64-unknown-linux-gnu             |
| rust_version       | rustc 1.66.1 (90743e729 2023-01-10)  |
| rust_channel       | stable-x86_64-unknown-linux-gnu      |
| cargo_version      | cargo 1.66.1 (ad779e08b 2023-01-10)  |
| pkg_version        | 0.74.0                               |
| build_time         | 2023-01-15 21:19:38 +00:00           |
| build_rust_channel | release                              |
| features           | database, default, trash, which, zip |
| installed_plugins  |                                      |

**Comment by fdncred** [comment]
`inc` is a plugin. If you're building from main, you'll have to build it with the build-all* or install-all* scripts. Then you have to register plugins.

**Comment by V0ldek** [comment]
I installed nushell via `cargo`.

```nu
cargo install nu
```

**Comment by fdncred** [comment]
you need to install the plugins that you want to use. the ones we include in our prebuilt binaries are also on crates.io like https://crates.io/crates/nu_plugin_inc

**Comment by V0ldek** [comment]
Thanks for that, it does work.

I guess I should just install `nu` through a different package manager then, I thought `cargo` was the safe bet, but if it doesn't have all the nooks then...

**Comment by fdncred** [comment]
you also didn't get the dataframe components which is installed with `cargo install nu --feature=dataframe`. the `nu_plugin_query` plugins can be especially helpful too for parsing web, xml, and json. There's also a good list of plugins here https://github.com/nushell/awesome-nu#plugins

**Comment by rgwood** [comment]
I agree with the general thrust of this issue. It's bad that we lead with examples that require plugins+installation. I think we should either rewrite the examples to use another command, or move `inc` into a built-in command.

I'll reopen this and move it to the docs repo.
